deepinv/datasets/mri/brainweb.py

```diff
@@ -0,0 +1,14 @@
+r"""A PyTorch dataset of 20 Normal Brains based on Brainweb"""
+
+from deepinv.datasets.base import ImageDataset
+import brainweb_dl as bwdl
```

Comment:
this will have to be an optional dependency, i.e. only gets imported when `download=True` in the dataset